packages/core/src/Serialization/PlutusData/PlutusData.ts

```diff
@@ -370,7 +370,43 @@ export class PlutusData {
    * @returns The Uint8Array or undefined if it can not be 'down cast'.
    */
   asBoundedBytes(): Uint8Array | undefined {
-    return this._bytes;
+    return this.#bytes;
+  }
+
+  /**
+   * Indicates whether some other PlutusData is "equal to" this one.
+   *
+   * @param other The other object to be compared.
+   * @returns true if objects are equals; otherwise false.
+   */
+  // eslint-disable-next-line complexity
+  equals(other: PlutusData): boolean {
```

Comment:
Even though this is indirectly tested in `PlutusMap.get`, this is a public method that deserves a couple of direct tests